app/controllers/ManageGroupTeamMembersController.scala

```diff
@@ -29,6 +29,7 @@ import services.{GroupService, SessionCacheService, TeamMemberService}
 import uk.gov.hmrc.agentmtdidentifiers.model._
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import views.html.groups._
+import views.html.groups.create.members.update_paginated_team_members
```

Comment:
can the new template go in manage instead of create?

eg. views.html.groups.manage.members.update_paginated_team_members 
may need a new package